### PR TITLE
156 fix portrait orientation for app and all for videos

### LINKED ISSUE #156: UI FIX - Fix Portrait Orientation for App and All for Videos
App should have fixed portrait orientation for all screens **minus** the video screen, where it should support all orientations

### PR DESCRIPTION
**Pull request by CodeGlitch**
Added fixed orientation on application start
Removes fixed orientation on VideosPage
Adds fixed orientation on VideosPage dispose
possibly closes #156

### CODE REVIEW

File: lib/ui/screens/videos/videos_page.dart [hunk 1]
```diff
@@ -35,11 +36,26 @@ class VideosPage extends BasePage {
...
+    ]);
+    super.dispose();
+  }
+
   /// For the initial list of faqs
   List<VideoModel> _videos = [];
 
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setPreferredOrientations([
```
Comment (Vanethos) [inline lib/ui/screens/videos/videos_page.dart]: maybe put this only in the video_player_page.dart!

Comment (CodeGlitch) [reply lib/ui/screens/videos/videos_page.dart]: Moved to the right file, thank you!

File: lib/ui/screens/video_player/video_player_page.dart [hunk 2]
```diff
@@ -22,24 +31,29 @@ class _VideoPlayerPageState extends State<VideoPlayerPage> {
...
+                controller: _controller,
+                onReady: () {
+                  _controller.addListener(() {});
+                },
+              ),
+            ),
+            Align(
+              alignment: Alignment.topRight,
+              child: Padding(
+                padding: const EdgeInsets.all(marginVidCloseBt),
```
Comment (Vanethos) [inline lib/ui/screens/video_player/video_player_page.dart]: Missing circular background with semi-transparent black

File: lib/ui/screens/video_player/video_player_page.dart [hunk 3]
```diff
@@ -22,24 +30,26 @@ class _VideoPlayerPageState extends State<VideoPlayerPage> {
...
-          controller: _controller,
-          onReady: () {
-            _controller.addListener(() {});
-          },
+      body: SafeArea(
+        child: Stack(
+          children: <Widget>[
+            Align(
+              alignment: Alignment.topRight,
+              child: IconButton(
```
[the remark this answers is not in the document]
Comment (CodeGlitch) [reply lib/ui/screens/video_player/video_player_page.dart]: Should be Added on 8f16b40

File: lib/ui/screens/video_player/video_player_page.dart (hunk 2 above)
Comment (CodeGlitch) [reply lib/ui/screens/video_player/video_player_page.dart]: Should be Added on 8f16b40

**Pull request merged**